web/docs/integrations/zeromq/README.md

```diff
@@ -0,0 +1,43 @@
+# ZeroMQ
+
+[ZeroMQ](https://zeromq.org/) (0mq) is a light-weight messaging framework with
+various socket types. Tenzir supports writing to [PUB
+sockets](https://zeromq.org/socket-api/#pub-socket) and reading from [SUB
+sockets](https://zeromq.org/socket-api/#sub-socket), both in server (listening)
+and client (connect) mode.
+
+![ZeroMQ](zeromq.svg)
+
+Use the IP address `0.0.0.0` to listen on all available network interfaces.
+
+Because ZeroMQ is entirely asynchronous, publishers send messages even when no
+subscriber is present. This can lead to lost messages when the publisher begins
+operating before the subscriber. To avoid data loss due to such races, pass
+`monitor=true` to activate message buffering until at least one remote peer has
+connected.
+
+:::tip URL Support
+The URL scheme `zmq://` dispatches to
+[`load_zmq`](../../tql2/operators/load_zmq.md) and
+[`save_zmq`](../../tql2/operators/save_zmq.md) for seamless URL-style use via
+[`from`](../../tql2/operators/from.md) and [`to`](../../tql2/operators/to.md).
+:::
```

Comment:
Does this also support `inproc://`, @IyeOnline?

Reply:
It does now 😉

Reply:
So we have `inproc` and `zmq` as schemes? Feels a bit weird. I don't see the need to abuse 0mq inproc pipelines if we can use `publish` and `subscribe`.

Reply:
`load_zmq` and `save_zmq` support both, so it's only natural to also support both in `from` and `to`.

Reply:
This goes in the opposite direction. I'd rather disable in the respective operators. We shouldn't allow this anymore; there is no valid 0mq use within a Tenzir process since we have `publish` and `subscribe`.

Reply:
I got no clue about ZeroMQ, but would `inproc` work with forked pipelines? If not, that's another argument to remove it. But in any case, it would be easy to add it back if it turns out that there is some need for it.

Reply:
Can we just do nothing and leave it undocumented? Or should we actually build it back?